### ISSUE TITLE
What is the name of the loss function used in the NMT?

### ISSUE THREAD

**Issue by code-rex1** [tensorflow/nmt#487]
What is the name of the loss function that is used?

Can anyone please help?

**Comment by Lapatou** [comment]
İt should be cross entropy
